testing/feedpost_test.go

```diff
@@ -91,6 +91,7 @@ func TestFeedPostParse(t *testing.T) {
 				"$type": "app.bsky.embed.images",
 				"images": [
 					{
+						"$type": "",
```

Comment:
Huh. This doesn't look right to me, it should either have a value or be omitted.

I wouldn't think that `EmbedImages_Image` ever needs `$type`, because it is not a top-level record type, and is not included in an enum itself. `EmbedImages` is in an enum, so it does need `$type` in that context.

Reply:
Yeah, agreed. Will investigate (by the same token i think the empty alt field seems wrong to me)